### PR TITLE
S5 frontend view refactor

### PR DESCRIPTION
**Pull request by NunoDasNeves**
little frontend refactor, fixes some inconsistencies and cleans tings up a little bit

### CODE REVIEW

File: frontend/src/App.vue [hunk 1]
```diff
@@ -46,6 +46,13 @@ html, body {
   --font-small-mobile: 12px;
 }
 
+.form-success {
```
Comment (DarkPurple141) [inline frontend/src/App.vue]: pretty sure these still don't do anything :-/

File: frontend/src/components/comments/CommentForm.vue [hunk 2]
```diff
@@ -5,7 +5,7 @@
           <p>
             <textarea placeholder="Your input here.." v-model="body"></textarea><br>
             <!-- should probs be a separate component -->
-            <AppButton @click.native="$emit('submitCommentForm', {body})">{{ type }}</AppButton>
+            <AppButton @click.native="callback({body})">{{ type }}</AppButton>
```
Comment (DarkPurple141) [inline frontend/src/components/comments/CommentForm.vue]: yay!

File: frontend/src/components/questions-answers/QuestionForm.vue [hunk 3]
```diff
@@ -5,8 +5,7 @@
           <p>
             <input type="text" placeholder="Question title..." v-model="title"><br>
             <textarea placeholder="Your question here.." v-model="body"></textarea><br>
-            <!-- should probs be a separate component -->
-            <AppButton @click.native="$emit('submitQuestionForm', {title, body})">Ask</AppButton>
+            <AppButton @click.native="callback({title, body})">Ask</AppButton>
```
Comment (DarkPurple141) [inline frontend/src/components/questions-answers/QuestionForm.vue]: Double yay.

File: frontend/src/components/questions-answers/QuestionForm.vue [hunk 4]
```diff
@@ -29,6 +28,9 @@ export default {
       title: '',
       body: ''
     }
+  },
+  props: {
+    callback: Function
```
Comment (DarkPurple141) [inline frontend/src/components/questions-answers/QuestionForm.vue]: ooft.

File: frontend/src/router.js [hunk 5]
```diff
@@ -31,13 +33,13 @@ export default new Router({
       props: ({query: { cid }}) => ({
```
Comment (DarkPurple141) [inline frontend/src/router.js]: is this dying tho? `cid` that is ?

File: frontend/src/router.js [hunk 6]
```diff
@@ -47,13 +49,13 @@ export default new Router({
...
```
Comment (DarkPurple141) [inline frontend/src/router.js]: needs to be updated.

**Comment by DarkPurple141** [comment]
Resolves #155.

File: frontend/src/router.js (hunk 5 above)
Comment (NunoDasNeves) [reply frontend/src/router.js]: Yes it's dying; out of scope of this issue though

File: frontend/src/router.js (hunk 6 above)
Comment (NunoDasNeves) [reply frontend/src/router.js]: will be in nexto branch i sposeo

**Pull request merged**